### PR TITLE
Fix flaky hosted service test

### PR DESCRIPTION
**Pull request by cailyncodes**

### CODE REVIEW

**Comment by jlhogan** [comment]
I'm not against this solution but would it be less error prone to make the `SetupHostedServices` method virtual, then override it in an extension for teststartup ?

**Comment by cailyncodes** [comment]
I see that point and could be easily convinced that is the better approach. My thought this explicitly writing everything out is that it requires intentionality about what we are including in our test env and helps limit a type of bug similar to this by having a code change directly to this file included.

File: test/HedwigTests/Fixtures/TestStartup.cs [hunk 1]
```diff
@@ -11,9 +14,54 @@ public class TestStartup : Startup
...
+			services.ConfigureFilters();
+			services.ConfigureControllers();
+			services.ConfigureSpa();
+			services.ConfigureMapping();
+			services.ConfigureRepositories();
+			services.ConfigureAuthentication(Configuration.GetValue<string>("WingedKeysUri"));
+			services.ConfigureAuthorization();
+			services.ConfigureValidation();
+			services.AddSingleton<IDateTime, SystemDateTime>();
+			services.AddSwaggerGen(c =>
```
Comment (cailyncodes) [inline test/HedwigTests/Fixtures/TestStartup.cs:31]: Also perhaps we should pull this out as a `ConfigureSwagger`

**Comment by jlhogan** [comment]
>My thought this explicitly writing everything out is that it requires intentionality about what we are including in our test env and helps limit a type of bug similar to this by having a code change directly to this file included.

SOLD ! :) extracting into a configure swagger also gets a big +!1 from me

**Pull request merged**